.github/workflows/web.yml

```diff
@@ -567,7 +567,7 @@ jobs:
         uses: ./.github/actions/package-nfpm
         with:
           nfpm_file_pattern: "centreon/packaging/*.yaml"
-          module_name: centreon-web
+          module_name: centreon
```

Comment:
you may want to avoid using module_name to target directories, as the "module_name" is used across many places and could either lead to confusion, or break packaging / packaging caches

Reply:
module_name isn't used anywhere in the package-nfpm action besides in upload-artifacts stuff, in that particular context that's perfectly fine